File: src/codes/Users_Manual/gain/transform_0.cs
~using System;
~using AUTD3Sharp;
~using AUTD3Sharp.Gain;
~using static AUTD3Sharp.Units;
var g = new Uniform(EmitIntensity.Max).WithTransform(dev => (tr, d) =>
{
    d.Intensity = d.Intensity / 2;
    d.Phase = new Phase((d.Phase.Radian + MathF.PI) * rad);
    return d;
});

File: src/codes/Users_Manual/gain/transform_0.cpp
//~#include<autd3.hpp>
//~#include <limits>
//~int main() {
const auto g =
    autd3::gain::Uniform(std::numeric_limits<autd3::EmitIntensity>::max())
        .with_transform([](const auto& dev, const auto& tr, auto d) -> auto {
          d.intensity = d.intensity / 2;
          d.phase = autd3::Phase((d.phase.radian() + autd3::pi) * autd3::rad);
          return d;
        });
//~return 0; }